packages/webapp/src/components/AddFarm/index.js

```diff
@@ -36,8 +32,6 @@ export default function PureAddFarm({
       <PageTitle onGoBack={onGoBack} title={title} style={{ marginBottom: '20px' }} />
       <Input style={style} {...inputs[0]} />
       <Input style={style} {...inputs[1]} />
-      <Input style={hiddenStyle} readOnly {...inputs[2]} />
-      <Input style={hiddenStyle} readOnly {...inputs[3]} />
```

Comment:
hookform register can work without attaching a input

Reply:
Cool. The docs said something about virtual inputs, but I couldn't figure out what it meant. This might be it.